lib/cog_api/http/old.ex

```diff
@@ -63,7 +63,7 @@ defmodule CogApi.HTTP.Old do
         %{"id" => id} ->
           {:ok, id}
         nil ->
-          {:error, %{"error" => "Resource not found"}}
+          {:error, %{"errors" => "Resource not found"}}
```

Comment:
Yes, good point. We can our error handling at a later point. This was to be able to get a baseline for now so I can move forward faster with the API changes and tests. We aren't merging this for now. I'll open an issue to handle multiple errors by setting ":errors" as a list in the struct.